### PR TITLE
Add @tangleid/jsonld package

### PR DESCRIPTION
**Pull request by ender503**
To use the jsonld.js easily and customize document loader, add
"@tangleid/jsonld" package.

Since the context URL definitions are used by "@tangleid/credential"
and "@tangleid/jsonld", consolidate the context definitions to
"packages/contexts.ts".

### CODE REVIEW

**Comment by jserv** [comment]
Add file `LICENSE` which describes copyright holder and terms.

File: packages/jsonld/README.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+```shell
+yarn add @tangleid/jsonld
+```
+
+## API Reference
+
+
+* [jsonld](#module_jsonld)
+
```
Comment (jserv) [inline packages/jsonld/README.md]: Be aware of the indention.

Comment (ender503) [reply packages/jsonld/README.md]: This section is automatically generated by `jsdoc2md`.

**Comment by ender503** [comment]
Append the `LICENSE` file. 👌

**Pull request merged**